src/components/Admin/UserPage/DownloadData/Util.tsx

```diff
@@ -47,6 +59,64 @@ interface InputProps {
   setValue: React.Dispatch<React.SetStateAction<number>>;
 }
 
+/**
```

Comment:
Request: please move the SensorInput stuff into a Util.tsx file in the main folder for ManageSensors because I'll use it in the delete sensor PR!